packages/admob/android/src/main/java/io/invertase/firebase/admob/ReactNativeFirebaseAdMobCommon.java

```diff
@@ -39,6 +44,34 @@
 
 public class ReactNativeFirebaseAdMobCommon {
 
+  static AdSize getAdSizeForAdaptiveBanner(ReactViewGroup reactViewGroup) {
+
+    try {
+      Display display = Objects.requireNonNull(((ReactContext) reactViewGroup.getContext()).getCurrentActivity()).getWindowManager().getDefaultDisplay();
+
+      DisplayMetrics outMetrics = new DisplayMetrics();
+      display.getMetrics(outMetrics);
+
+      float widthPixels = outMetrics.widthPixels;
+      float density = outMetrics.density;
+
+      int adWidth = (int) (widthPixels / density);
```

Comment:
```suggestion
      int adWidth = (int) (outMetrics.widthPixels / outMetrics.density);
```

no need for temporary variables if only used once